projects/gstreamer/build.sh

```diff
@@ -0,0 +1,110 @@
+#!/bin/bash -eu
+# Copyright 2017 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+echo "CFLAGS" $CFLAGS
+echo "CXXFLAGS" $CXXFLAGS
+export LDFLAGS="$SANITIZER_FLAGS $COVERAGE_FLAGS"
```

Comment:
I don't think COVERAGE_FLAGS is a part of our interface.

Reply:
Looks legit https://github.com/google/oss-fuzz/blob/master/infra/base-images/base-builder/README.md#build-configuration

Reply:
My bad, looked at the wrong section: https://github.com/google/oss-fuzz/blob/master/infra/base-images/base-builder/README.md#compiler-flags

Reply:
So which is it ? Good or not good ? :)

Reply:
Not good :) 
Just use `LDFLAGS="$CXXFLAGS"`

Reply:
We actually didn't need a custom LDFLAGS. Removed it altogether